Created workflow to build and push ubi-ppc64le image to GHCR

Shafi-Hussain opened the pull request:
Added a workflow to build and push ubi-ppc64le images to GitHub Container Registry.

Shafi-Hussain committed:
Created workflow to build and push ubi-ppc64le image to GHCR

mkumatag commented on .github/workflows/ubi-ppc64le-build.yml:
```diff
@@ -0,0 +1,50 @@
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: 'ubi-ppc64le'
```
this is not just enough, 
```suggestion
  IMAGE_NAME: ${{ github.repository_owner }}/'ubi-ppc64le'
```

so that it will get published to `ghcr.io/osuosl/ubi-ppc64le`

Shafi-Hussain replied:
Thanks, I missed it.
I have updated the workflow with the owner

mkumatag commented on .github/workflows/ubi-ppc64le-build.yml:
```diff
@@ -0,0 +1,50 @@
+                context: "{{defaultContext}}:ubi-ppc64le"
+                platforms: linux/ppc64le
+                tags: ${{ steps.meta.outputs.tags }}
+                labels: ${{ steps.meta.outputs.labels }}
```
add a newline in the end to get rid of this `(-)`

Shafi-Hussain committed:
Update ubi-ppc64le-build.yml

Added missing repository owner for IMAGE_NAME

Shafi-Hussain committed:
Update ubi-ppc64le-build.yml

ramereth committed:
Update to match how we built the other image

Standardized how we build and test this and adds a schedule to rebuild weekly.

Signed-off-by: Lance Albertson <lance@osuosl.org>

mkumatag commented:
@ramereth ghcr.io/osuosl/ubi-ppc64le:latest image pushed, probably it is pushed as a private(by default) one, can you make this public so that we all can consume?

ramereth commented:
@mkumatag give it a try now. Should be open

mkumatag commented:
> @mkumatag give it a try now. Should be open

perfect, I can see the image and pull as well. Thanks

```
[root@mkumatag-ws ~]# podman run -ti ghcr.io/osuosl/ubi-ppc64le:latest uname -a
Trying to pull ghcr.io/osuosl/ubi-ppc64le:latest...
Getting image source signatures
Copying blob a1a95a3515ab done   |
Copying blob c792e30d0a64 done   |
Copying blob fd06d38f6d47 done   |
Copying blob 977e55c9aee0 done   |
Copying blob e1e4ca27d345 done   |
Copying blob c1167695d0a5 done   |
Copying blob 060ed989d00c done   |
Copying blob 5910532ce120 done   |
Copying blob caea0b7a0374 done   |
Copying blob 6ffa95a9581b done   |
Copying blob 4f4fb700ef54 skipped: already exists
Copying blob f98a63c0d497 done   |
Copying config d4338cfcf8 done   |
Writing manifest to image destination
+ [[ '' == ssh-* ]]
+ [[ 2 -gt 0 ]]
+ [[ uname == ssh-* ]]
+ exec uname -a
Linux c341bb83d9cd 5.14.0-547.el9.ppc64le #1 SMP Mon Dec 30 19:53:09 UTC 2024 ppc64le ppc64le ppc64le GNU/Linux
[root@mkumatag-ws ~]#
```
